src/components/NotFound.jsx

```diff
@@ -4,6 +4,12 @@ const NotFound = () => {
   return (
     <div className="bg-witl-purple-400 min-h-screen font-marcellus-sc">
       404 Page Not Found
+      <p
+        className="text-witl-white-100 text-4xl mb-8 flex flex-col items-center justify-center"
+        style={{ paddingBottom: "60px" }}
```

Comment:
use tailwind class utility --- such as pb-12 or something like that find a value that works. pb stands for padding bottom. check the tailwind website for useful classes

Reply:
made changes to use tailwind class utility